samples/boards/stm32/stm32l1_disco/pm_rtc/prj.conf

```diff
@@ -0,0 +1,20 @@
+#overwrite board config, since use of 'c' version
+CONFIG_SOC_STM32L151XB=n
+CONFIG_SOC_STM32L152XC=y
```

Comment:
This might be requested on your board, but can't go upstream

Reply:
Is there a way to still allow this? Or to find another work-around regarding the board SoC? 151XB SoCs don't have subsecond support, so this is required for the STM32 RTC timer driver to work (per current implementation).

Reply:
How about a board overlay? [docs link](https://docs.zephyrproject.org/latest/guides/dts/howtos.html#set-devicetree-overlays)

Reply:
Maybe another option would be to implement something similar to what has been done for the nucleo_l452re board, since I see there that two 'types' are included, and depending on the type the correct configs+dts files are provided.